{
  "uid" : "53d1f3f1f7307e08",
  "name" : "test_broken",
  "fullName" : "test_allure#test_broken",
  "historyId" : "985c3d972d231f87d0914b89f78754fd",
  "time" : {
    "start" : 1672927915565,
    "stop" : 1672927915565,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "Exception: oops",
  "statusTrace" : "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure.py:19: Exception",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "test_allure"
  }, {
    "name" : "host",
    "value" : "Windows"
  }, {
    "name" : "thread",
    "value" : "21712-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_allure"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "26b4e85a596b49ff",
      "status" : "broken",
      "statusDetails" : "Exception: oops",
      "time" : {
        "start" : 1672927915565,
        "stop" : 1672927915565,
        "duration" : 0
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "53d1f3f1f7307e08.json",
  "parameterValues" : [ ]
}